hack/latest-kind-images.sh

```diff
@@ -47,7 +47,7 @@ EOF
 
 curl -fsSL "https://api.github.com/repos/kubernetes-sigs/kind/releases/tags/${kind_version}" \
     |  jq -r '
-[ .body  | capture("- 1\\.(?<minor>[0-9]+): `kindest/node:v(?<version>[^@]+)@sha256:(?<sha256>[^`]+)`\r"; "g") ]
+[ .body  | capture("- v?1\\.(?<minor>[0-9]+)(.(?<patch>[0-9]+))?: `kindest/node:v(?<version>[^@]+)@sha256:(?<sha256>[^`]+)`\r"; "g") ]
```

Comment:
A change in the release notes format for this release:
 * https://github.com/kubernetes-sigs/kind/releases/tag/v0.21.0